src/java/ognl/ListPropertyAccessor.java

```diff
@@ -244,7 +244,7 @@ public String getSourceSetter(OgnlContext context, Object target, Object index)
                 
                 if (m != null || !context.getCurrentType().isPrimitive())
                 {
-                    System.out.println("super source setter returned: " + super.getSourceSetter(context, target, index));
+                    // System.out.println("super source setter returned: " + super.getSourceSetter(context, target, index));
```

Comment:
Maybe throw it away?

Reply:
I'm not sure if it should be thrown away, since there are a bunch of similar commented-out println() calls throughout OGNL ?

I think the statement (and similar ones elsewhere) are intended to be "uncommented locally" when debugging the "OGNL internals".  It's quicker to uncomment-and-recomment than have to re-create them ... but I could be mistaken.  :smile:

Reply:
👍